board/fischertechnik/TXT/rootfs/etc/sudoers.d/display-settings

```diff
@@ -2,4 +2,4 @@
 ## for changing display settings
 
 Defaults!/usr/bin/ts_calibrate env_keep=TSLIB_TSDEVICE
-ftc     ALL = NOPASSWD: /usr/bin/ts_calibrate, /etc/init.d/S90launcher restart
+ftc     ALL = NOPASSWD: /usr/bin/ts_calibrate, /etc/init.d/S90launcher restart, /sbin/set-touchscreen-calibration-reset-flag, /sbin/unset-touchscreen-calibration-reset-flag
```

Comment:
Do you really need new sudo scripts here just to set and reset the touchscreen flag?

Alternate suggestion: Put the flag some place where the ftc user can write anyway (e.g. in `/home/ftc/.touchscreen-calibration`). The startupt script (run as root) can read from there just fine, and the extra scripts allowing ftc to write the file are not necessary.

Reply:
I would keep it root because the tslib calibration is system-wide

Reply:
As to the calibration data itself: Agreed. But that is written by `ts_calibrate`, which already runs as root.

But the calibration _GUI tool_ is a different matter - _that_ will in all probability only ever be run from the ftc account, so there is no need to keep the tool specific data (i.e. the "restore calibration on reboot" flag) in `/etc`.  And in fact, it _can_ only be run by ftc right now, at least without prompting for a sudo password (and your current PR does not change this in any way).

So, there really is no need to keep the "reset the calibration data on reboot" flag in `/etc/` IMO.

Reply:
Agreed